### DISCUSSION TITLE
Missing Bladeview and Bladeitem

### DISCUSSION THREAD

**Discussion by Ch3my** [CommunityToolkit/Windows#583] [Q&A] [answered]
Looks like there is no way to have a BladeView, is not listed on the announcement and I can't find a package that includes Bladeview. Is there any substitution?

**Comment by zhuxb711** [comment]
They just did not port it into the new repo and the priority is low

**Reply by michael-hawker** [reply, marked as answer]
Yup, has not been prioritized yet. See issue #89, open for anyone who wants to take it on. There's just been a lot of design and modernization discussion that needs to be correlated to move forward on it.
